ckanext/aberdeen/actions.py

```diff
@@ -233,18 +253,68 @@ def send_inactive_users_email(context, data_dict):
     '''Sends the inactive users list to all site system administrators'''
 
     user_list = ckan.logic.get_action('user_list')(context, data_dict)
-    admin_email_list = [
-        user['email'] for user in user_list if user['sysadmin'] is True]
-
+    admin_list = [user for user in user_list if user.get('sysadmin') is True]
     inactive_users = toolkit.get_action('inactive_users')(context, data_dict)
+    number_of_users = len(inactive_users)
 
-    # Format the inactive users list for better readability in an email
-    inactive_users = '\n\n'.join(
-        '\n'.join(info + ': ' + str(user_info[info]) for info in user_info)
-        for user_info in inactive_users)
-
-    for admin_email in admin_email_list:
-        if admin_email:
+    if not inactive_users:
+        inactive_users = \
+            'Hi {},\n\nThere are no inactive users to report this week.'
+    else:
+        inactive_users = \
+            'Hi {},\n\nThere are currently {} inactive users:\n\n'
+
+        # Format the inactive users list for better readability in an email
+        for user in user_list:
+            display_name = user.get('display_name')
+            name = user.get('name')
+            profile_url = '{}/user/{}'.format(
+                config.get('ckan.site_url'), name)
+            id = user.get('id')
+            last_activity = user.get('last_activity')
+            created = user.get('created')
+            full_name = user.get('fullname')
+            email_address = user.get('email')
+            sysadmin = user.get('sysadmin')
+            state = user.get('state')
+
+            inactive_users += """
+            Display name: {}
+            Profile URL: {}
+            ID: {}
+            Last activity date: {}
+            Account creation date: {}
+            Username: {}
+            Full name: {}
+            Email address: {}
+            System administrator: {}
+            State: {}\n
```

Comment:
We don't need this much info. Let's keep it simple... Remove following
- Display name: this is same as full name
- ID: no need
- username: present in URL
- System administrator: no need
- state: no need

Reply:
Yeah, I wasn't sure how much detail they would want. I had already omitted a couple of fields, but you're right - these are probably unnecessary.